src/lib/utils.js

```diff
@@ -232,6 +232,7 @@ function getRefinements(results, state, clearsQuery) {
             type: 'query',
             name: state.query,
```

Comment:
Do we want to display `"my query"` instead of `my query`?

If so, it'd be nice to wrap the `name` value with double quotes.

Reply:
ah yes, we've mentioned this in [the related PR I made on ais](https://github.com/algolia/angular-instantsearch/pull/301#discussion_r226354923)
Isn't it a `<q>...</q>` tag IIUC ?

Reply:
Yep, should be done in a q tag which displays as quotes

Reply:
@tkrugg It should be in this file then: https://github.com/algolia/instantsearch.js/blob/develop/src/widgets/current-refined-values/defaultTemplates.js

Reply:
![image](https://user-images.githubusercontent.com/2982512/47362267-56a70f80-d6d4-11e8-824a-dd901edc5bbe.png)

So I can't find where that template rendering is tested. If I read correctly the render method is mocked so changing the default template can't break the tests?

Reply:
You can test the rendering in the component itself when rendering a `query`: https://github.com/algolia/instantsearch.js/blob/1510a9439bb22dd31eb4dc0c75bd4316a57db73d/src/components/CurrentRefinedValues/__tests__/CurrentRefinedValues-test.js

Reply:
I maybe be missing something but this doesn't seem to be testing the default template neither since it's mocked.
https://github.com/algolia/instantsearch.js/blob/1510a9439bb22dd31eb4dc0c75bd4316a57db73d/src/components/CurrentRefinedValues/__tests__/CurrentRefinedValues-test.js#L16-L23

Reply:
Right, then getting back to [the widget test file](https://github.com/algolia/instantsearch.js/blob/1510a9439bb22dd31eb4dc0c75bd4316a57db73d/src/widgets/current-refined-values/__tests__/current-refined-values-test.js), you can pass the [default template](https://github.com/algolia/instantsearch.js/blob/1510a9439bb22dd31eb4dc0c75bd4316a57db73d/src/widgets/current-refined-values/defaultTemplates.js) to your widget (it's already imported).

Reply:
I ended up testing the defaultTemplate separately which I think is more consistent with the ways the concerns have been separated up to now. What do you think?